src/runtime/auto-prompt-manager.js

```diff
@@ -673,4 +691,22 @@ export class AutoPromptManager {
     }
     return true;
   }
+
+  /**
+   * Checks if provided ExperimentFlag is returned in article endpoint.
+   * @param {?./entitlements-manager.Article|undefined} article
+   * @param {string} experimentFlag
+   * @return {!Promise<boolean>}
+   */
+  async isExperimentEnabled_(article, experimentFlag) {
+    if (article === undefined) {
```

Comment:
Just curious, could we simplify this to return `false` if the `article` param is falsy?